Storage/Bookmarks/BookmarksModel.swift

```diff
@@ -62,6 +62,11 @@ public class BookmarkFolder: BookmarkNode {
     override public var canDelete: Bool {
         return false
     }
+
+    public func removeItemWithGUID(guid: GUID) -> MemoryBookmarkFolder {
```

Comment:
I'm not sure it's worth putting this implementation here rather than in `MemoryBookmarkFolder`. I just noticed that the `subscript` implementation is already a stub, so there's no advantage to making this abstract class do anything.

Indeed, I wonder if the language constraint that forced us to make `BookmarkFolder` a class rather than a protocol still applies?